assets/src/edit-story/components/panels/textStyle/font.js

```diff
@@ -101,11 +90,10 @@ function FontControls({ selectedElements, pushUpdate }) {
             <DropDown
               data-testid="font.weight"
               ariaLabel={__('Font weight', 'web-stories')}
+              placeholder={__('(multiple)', 'web-stories')}
```

Comment:
Nope.

Any text selection, even in a single text field, can have more than one font weight. And will only show the placeholder specifically in case of "multiple". "Multiple" here does not mean "multiple text fields with different values", but just "multiple font weights present in selection", which can be anything from a selection inside a text field, to an entire text field to multiple text fields.